libc/spec/spec.td

```diff
@@ -55,6 +55,23 @@ def UnsignedShortType : NamedType<"unsigned short">;
 // TODO: Add compatibility layer to use C23 type _Float128 if possible.
 def Float128Type : NamedType<"float128">;
 
+// Fixed point types.
```

Comment:
these type definitions should be in stdc_ext.td

Reply:
Done.